database/datagenerator.js

```diff
@@ -82,7 +18,11 @@ function generateReviewCsv(writer, encoding, callback) {
       id += 1;
       if (id % 1000000 === 0) { console.log(id); }
       const propertyId = randomNum(10000000);
+      const hostPicture = faker.image.avatar();
```

Comment:
I think it's awesome that you're adding a few more random generations here to make the data more interesting.